var/spack/repos/builtin/packages/bcftools/package.py

```diff
@@ -33,8 +49,59 @@ class Bcftools(AutotoolsPackage):
     depends_on('htslib@1.3.1', when='@1.3.1')
     depends_on('htslib@1.2', when='@1.2')
 
+    patch('makefile_12.patch', when='@1.2')
+    patch('fix_mk.patch', when='@1.2')
+    patch('makefile_13.patch', when='@1.3')
+    patch('makefile_14.patch', when='@1.4')
+    patch('guess-ploidy.py_2to3.patch', when='@1.6:1.9')
+
+    @when('@1.5:')
     def configure_args(self):
         args = []
+
         args.append('--with-htslib={0}'.format(self.spec['htslib'].prefix))
+        args.extend(self.enable_or_disable('libgsl'))
+
+        if self.spec.satisfies('@1.8:'):
+            args.extend(self.enable_or_disable('perl-filters'))
 
         return args
+
+    @when('@1.2:1.4')
+    def set_make_options(self):
+        options = []
+
+        options.append('prefix={0}'.format(self.prefix))
+        options.append('HTSDIR={0}'.format(self.spec['htslib'].prefix))
+
+        if '+libgsl' in self.spec:
+            options.append('USE_GPL=1')
+
+        return options
+
+    @when('@1.2:1.4')
+    def autoreconf(self, spec, prefix):
+        touch('configure')
+
+    @when('@1.2:1.4')
+    def configure(self, spec, prefix):
+        pass
+
+    @when('@1.2:1.4')
+    def build(self, spec, prefix):
+        make_options = self.set_make_options()
+        make(*make_options)
+
+    @when('@1.2:1.4')
+    def install(self, spec, prefix):
+        make_options = self.set_make_options()
+        make(*make_options, 'install')
+
```

Comment:
Travis is failing for Python 2 because of a syntax error around line 99. This is usually because of the presence of unicode, but I don't see anything fishy.

Reply:
I will take a look.

Reply:
I do not see anything there except an end of line marker.

Reply:
It might be off by a line or something.

Reply:
It is the line above it,
```
make(*make_options, 'install')
```
that needs to be,
```
make('install', *make_options)
```